app/assets/sass/application.scss

```diff
@@ -1,52 +1,65 @@
 // Path to assets for use with the file-url function
 // in the govuk frontend toolkit's url-helpers partial
 $path: "/public/images/";
-
 // This includes entire govuk-elements from _govuk-elements.scss
 @import 'govuk-elements';
-
 // Take a look at in app/assets/sass/patterns/ to see which files are imported.
 @import 'patterns/check-your-answers';
 @import 'patterns/unbranded';
-
 // Autocomplete styles from https://github.com/ministryofjustice/prison-visits-public
 @import 'moj.autocomplete';
+@mixin respond-to($media) {
+     @if $media == handhelds{
+        @media only screen and (max-width: 768px) {
+             @content;
+        }
+    }
+    @else if $media == medium-screens {
+         @media only screen and (min-width: 769px) and (max-width: 1024px){
+            @content;
+        }
+    }
+    @else if $media == wide-screens {
+         @media only screen and (min-width: 1024px){
+            @content;
+        }
+    }
+}
 
 button.link {
-  color: #005ea5;
-  text-decoration: underline;
-  font-size:16px;
-  line-height:24px;
-  background:none!important;
-  border:none;
-  padding:0!important;
-  cursor: pointer;
+    color: #005ea5;
```

Comment:
Use 2 space tabs, not 4. We are using 2 space tabs in rest of application for StandardJS and differences will cause issues when someone else edits this file.